src/nal/pps.rs

```diff
@@ -142,7 +142,8 @@ impl SliceGroup {
 
 #[derive(Debug, Clone)]
 pub struct PicScalingMatrix {
-    // TODO
+    pub scaling_list4x4: Vec<ScalingList<16>>,
+    pub scaling_list8x8: Vec<ScalingList<64>>,
```

Comment:
Do you think it'd be more clear to make this `Option<Vec<ScalingList<64>>` based on the `transform_8x8_mode_flag`, as opposed to representing this by `scaling_list8x8.empty()`? I could go either way; probably should have a comment in the latter case though.

Reply:
After some thinking, representing it as `Option<...>` makes more sense, it seems more intuitive if someone wants to extract the flag.